{{cookiecutter.project_slug}}/README.md

```diff
@@ -10,6 +10,14 @@
 
 ## Features
 
+{% if cookiecutter.include_src_folder in ["yes", "y"] -%}
+* src layout: Is a good practice moving the Python code that is
```

Comment:
as I mentioned src layout is not the most used ... probably you need to check your sources.
but currently, this layout is been adopted by many project

Reply:
could you share that link again?

Reply:
btw the link you have in the first comment here should be enough for getting content for our documentation: https://packaging.python.org/en/latest/discussions/src-layout-vs-flat-layout/

Reply:
Ok, I'll take it just from there. Thanks for the suggestion